CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 As this project is pre 1.0, breaking changes may happen for minor version bumps.
 A breaking change will get clearly notified in this log.
 
+## unreleased
+
+- Add Stellar Protocol 11 compatibility (breaking change)
+  - Rename `manage_offer` to `manage_sell_offer`.
+  - Rename `create_passive_offer` to `create_passive_sell_offer`.
+  - Add `manage_buy_offer`.
```

Comment:
Some things I noticed: 

* The breaking change warning isn't super visible. The changelog should have earlier updates that bold the words and put it at the beginning, I think it's worth maintaining that convention
* If I'm just a JS dev, I might not understand what implications this has to the API of the library, so it might make sense to frame this in terms of the library exports that are being added (I guess it's renaming `Operation.manageOffer` to `Operation.manageSellOffer`?)